frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -537,6 +540,7 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
 
                         posthog.capture('recording_snapshot_loaded', {
                             source: source.source,
+                            duration: Math.round(performance.now() - snapshotLoadingStartTime),
```

Comment:
Add timings for individual sources. Trying to understand which sources are the slowest to load.

Note: `snapshotLoadingStartTime` resets every time `loadRecordingSnapshots` is called, but `cache.snapshotsStartTime` does not as it measures the time the first snapshot source loading was started